agent/eventhandler/task_handler.go

```diff
@@ -256,3 +279,77 @@ func (handler *TaskHandler) SubmitTaskEvents(taskEvents *eventList, client api.E
 		})
 	}
 }
+
+// sendEvent tries to send an event, specified by 'eventToSubmit', of type
+// 'eventType' to ECS
+func sendEvent(sendStatusToECS sendStatusChangeToECS,
+	setChangeSent setStatusSent,
+	eventType string,
+	client api.ECSClient,
+	eventToSubmit *list.Element,
+	stateSaver statemanager.Saver,
+	backoff *utils.SimpleBackoff,
+	taskEvents *eventList) {
+
+	// Extract the wrapped event from the list element
+	event := eventToSubmit.Value.(*sendableEvent)
+	seelog.Infof("TaskHandler, Sending %s change: %s", eventType, event.String())
```

Comment:
This changed from the prior if blocks from `Debug` to `Info`. I like this increase in verbosity better than some of the other `Info` level additions. However, I might flip the level on to `Debug` and log only the success message (line 309) as `Info`.